23/README.md

```diff
@@ -0,0 +1,283 @@
+
+- **Accepted** by the network. In order to be accepted, a transaction must be "valid," where the definition of valid varies from network to network.
+- **Committed** in the blockchain. If a transaction makes it into a block, it is stored together with other transactions in the local database of the node.
+- **Finalized**. Finality means that once a transaction is committed, it cannot be reversed, i.e. the data cannot be rolled back to the previous state. Different blockchain systems may provide different types of finality. Typically, this is defined in the consensus protocol. Different types of consensus exist, such as voting-based consensus with immediate finality and lottery-based consensus with probabilistic finality. (Definition from the [Hyperledger Performance and Scale Working Group][hl:finality].) For BigchainDB, once a transaction is committed it is also finalized.
+
+Note that BigchainDB inherits Tendermint **instant finality**. This means that as soon as a new block is committed, it cannot be reversed. Through this document, we won't make any distinction between a *committed* transaction and a *finalized* transaction. For clarity, we will always prefer the term *finalized*.
+
+Given that, the metrics we measured in our experiments are:
+
+- Transaction acceptance rate.
+- Transaction finalization rate.
+- Time to accept a transaction (how much time it takes for a valid transaction to be stored in the mempool).
+- Time to finalize a transaction (how much time it takes for a valid transaction to be stored in the blockchain).
+
+Those metrics reflect the different states of a valid transaction: accepted and finalized.
+
+Even if the reference metric is how many transactions the network finalizes per second, during this analysis we dig deeper and analyze the distribution on accepted transactions, committed etc. We want to make sure that the system is responsive most of the time, and we don't want to get fooled by an average that hides a lot of data behind it. Knowing the actual distribution of how much time it takes for a transaction to be accepted and then committed allows us to have a clear understanding of the responsiveness of the system, and this is something we won't know if we look only at the average. For this reason we developed our own tool, `bigchaindb-benchmark`.
+[BigchainDB Benchmark][bdb:benchmark-tool] is the tool we developed to generate the workload and collect statistics. It can simulate a large amount of workload by spawning multiple processes, and output a CSV file to timestamp the different steps in the life cycle of a transaction. This allows us to measure the distribution of how long it takes for a transaction to be accepted and later finalized.
```

Comment:
These sentences are repetitive, i.e.

> For this reason we developed our own tool, `bigchaindb-benchmark`. [BigchainDB Benchmark][bdb:benchmark-tool] is the tool we developed to generate the workload and collect statistics.

They could be simplified into one sentence such as:

> For this reason we developed our own benchmarking tool to generate the workload and collect statistics; we call it [bigchaindb-benchmark][bdb:benchmark-tool].